### ISSUE TITLE
Add Participants in Sakai should show participants from all rosters added to the Sakai site

### ISSUE THREAD

**Issue by joliet** [DukeLearningInnovation/kits#448]
I am an instructor trying to add participants to my Sakai course site but I only see a subset of the students for all of the roster I have added to my site.  This makes me confused and concerned that not all students have access to my Sakai site.

How might we redesign how Sakai and Kits interact when updating participants *from Sakai*.

**Questions**
If instructors saw the View/Edit Roster screen in the current Combined Kits design, what would their response be?  Needs testing.
Is it acceptable when adding new participants to the course to have to select one of the sections for adding, as long as that person has access to all tools/areas of Sakai?

**Assumptions**

**Hypotheses**

Related to - https://github.com/DukeLearningInnovation/kits/issues/450 (Participant list in Toolkits/Kits and Sakai should always be the same)

**Comment by ewendland23** [comment]
**Multi-Roster Test Suite: (please make sure you use ALL new kits and Sakai sites.)**
- Check normal starting from Sakai:
  * Create Sakai Course site for current semester.
  * Add participants -> Add another instructor
  * Log into Kits (act as the real instructor).
    - Kit should exist
    - Roster should include new instructor
    - Sakai site should be linked properly

- Check multi from Sakai:
  * Create Sakai Course site for current semester
  * Add additional rosters
  * Add participants -> Add another instructor
  * Log into Kits (act as the real instructor).
    - Kit should exist as a Combined course with the sub-kits created
    - Roster should include new instructor
    - Sakai site should be linked properly to the Combined course (not to subkits)
  * Log into Kits (act as a student).
    - should see the proper subkit.
    - Sakai site should be linked.

  - Check multi from Sakai with no Add Participants (this forces the Kit to be created on login):
    * Create Sakai Course site for current semester
    * Add additional rosters
    * Log into Kits (act as the real instructor).
      - Initially the kits will all show up but not the Combo. Sakai site will be linked to first.
      - Later Kit should exist as a Combined course with the sub-kits created
      - Sakai site should be linked properly to the Combined course (not to subkits)
    * Log into Kits (act as a student).
      - should see the proper subkit.
      - Sakai site should be linked.

**Comment by joliet** [comment]
I created a Sakai site with multiple sections on sakai-test for Susan Rodger - COMPSCI 201 001 with sections 01D - 20D.  When trying to add participants to the Sakai site, I received a prompt that I needed to update the participant list in Kits. After clicking on Kits in the link, I was able to see the combined kit in Kits and update there.  The Sakai site showed up on all the kits and linked to the right Sakai site.

When I logged into Kits as one of the students in the combined kit, I did not see the Sakai site on the lecture and section kits.  The student I tested with is Brennan Simon (bgs23) and should see a Sakai site on the kits COMPSCI 201 - 001 Sp20 and COMPSCI 201 - 02D Sp20.

This was also the case with the last test case - create multi in Sakai with no add participants then go to Kits.  The combined kit created and the Sakai showed up on all cards with the right participants.  But when I viewed as a student, he did not see the lecture and section kits that he should have.  The course was ECON.101D.001.Sp20 and he should have seen the kit for that course and the section ECON.101D.03D.Sp20.
